src/App/components/Section/components/LocationBlock/LocationBlock.module.scss

```diff
@@ -99,12 +100,16 @@
       font-family: roboto-medium;
       font-size: $fs18;
       margin-bottom: $space-xxs;
-      width: 7.1rem;
       display: -webkit-box;
       -webkit-line-clamp: 2;
       -webkit-box-orient: vertical;
       overflow: hidden;
     }
+    .truncate {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
```

Comment:
We can move this piece of code to the index.scss file, so, we will be able to use .truncate class everywhere we want instead of creating it every time when it needs.